js/main.js

```diff
@@ -11,3 +12,441 @@ document.querySelectorAll('.a-links').forEach((n) => n
+    year: '2015',
+  },
+];
+
+// Create the desktop modal object with elements
+const desktopModalArray = [
+  { // card 1 details
+    cardTitle: 'Tonic',
+    id: 'd1',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. printer took a galley of type and scrambled it 1960s with the relea',
+    company: 'CANOPY',
+    cardImage: './assets/images/desktop1.png" alt="project1.png"',
+    counter1: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    role: 'Front End Dev',
+    skills: ['HTML', 'CSS', 'Javascript'],
+    counter2: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    year: '2015',
+    sourceLink: 'https://github.com/mercy/portfolio',
+    demoLink: 'https://mercymugambi.github.io/portfolio/',
+  },
+  // card2 details
+  {
+    cardTitle: 'Multi Post stories',
+    id: 'd2',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. printer took a galley of type and scrambled it 1960s with the relea',
+    company: 'Facebook',
+    cardImage: './assets/images/desktop2.png" alt="project2.png"',
+    counter1: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    role: 'Back End Dev',
+    skills: ['HTML', 'Ruby on rails', 'CSS', 'Javascript'],
+    counter2: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    year: '2015',
+    sourceLink: 'https://github.com/mercymugambi/portfolio',
+    demoLink: 'https://mercymugambi.github.io/portfolio/',
+  },
+  // card3 Details
+  {
+    cardTitle: 'Facebook 360',
+    id: 'd3',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. printer took a galley of type and scrambled it 1960s with the relea',
+    company: 'facebook',
+    cardImage: './assets/images/desktop3.png" alt="project3.png"',
+    counter1: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    role: 'Back End Dev',
+    skills: ['HTML', 'Ruby on rails', 'CSS', 'Javascript'],
+    counter2: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    year: '2015',
+    sourceLink: 'https://github.com/mercymugambi/portfolio',
+    demoLink: 'https://mercymugambi.github.io/portfolio/',
+  },
+  // card4 Details
+  {
+    cardTitle: 'Uber Navigation',
+    id: 'd4',
+    closeBtn: '&times;',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. printer took a galley of type and scrambled it 1960s with the relea',
+    company: 'Uber',
+    cardImage: './assets/images/desktop4.png" alt="project4.png"',
+    counter1: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    role: 'Back End Dev',
+    skills: ['HTML', 'Ruby on rails', 'CSS', 'Javascript'],
+    counter2: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    year: '2015',
+    sourceLink: 'https://github.com/mercymugambi/portfolio',
+    demoLink: 'https://mercymugambi.github.io/portfolio/',
+  },
+];
+
```

Comment:
Hi @mercymugambi 
Great joy so far implementing this project requirement up to this extent.

 The project has a problem haer
 

- [ ] Please we need to have a single array of objects to store JavaScript objects containing the information for each project.

**Let reason together**
Imagine this was a database table storing your project data, will you create multiple tables to store data at different screen-size? I strongly believe your answer must be `NO`
We need just one database table to store our project data with their different `id:` and write CSS to make our screen display [responsive](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)

- [ ] Please remember to refactor your logical code below